regression/goto-instrument/dump-enum/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--dump-c
+enum hex \{ .* \};
```

Comment:
This feels like a fairly minimal check given the contents of the test file.  Could this be strengthened or, alternatively, documented in the test case why this is sufficient.

Reply:
Oh, indeed what was obvious to me as I saw the failures first-hand wasn't at all obvious here! I've added a comment to the test.desc file.

Reply:
Right!  I get it now!